test/functional/apps/visualize/_point_series_options.js

```diff
@@ -203,11 +204,90 @@ export default function ({ getService, getPageObjects }) {
 
         const labels = await PageObjects.visualize.getXAxisLabels();
 
-        await kibanaServer.uiSettings.replace({ 'dateFormat:tz': 'Browser' });
+        expect(labels).to.eql(expectedLabels);
+      });
+
+      it('should show different labels in different timezone', async function () {
+        const fromTime = '2015-09-22 09:05:47.415';
+        const toTime = '2015-09-22 16:08:34.554';
+        // note that we're setting the absolute time range while we're in 'America/Phoenix' tz
+        await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
+        await PageObjects.visualize.waitForRenderingCount();
```

Comment:
We mostly use `waitForVisualization` in similar cases, because `waitForRenderingCount()` just checks that rendering count >=1. `waitForVisualization` checks that rendering count is constant within 2 sec.

```
        await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
        await PageObjects.visualize.waitForVisualization();
        await PageObjects.discover.brushHistogram(0, 1);
        await PageObjects.visualize.waitForVisualization();
```